src/Bank.java

```diff
@@ -0,0 +1,8 @@
+public class Bank {
+    private long balance;
+
+    public long add(Transaction transaction) {
+        balance = balance + transaction.getMoney();
```

Comment:
Code for updating a variable using `+` has an even shorter form than `a = a + x`, which is `a += x`.
In other words, we can simplify this to be `balance += transaction.getMoney();`
This also works for other mathematical operations (`*`, `/`, `-` becomes `*=`, `/=`, `-=`), and even applies to string concatenation:
```java
private String completeError = "";

String augmentErrorReport(String newestError) {
    completeError += newestError;
    return completeError;
}